exercises/practice/square-root/.meta/example.rkt

```diff
@@ -0,0 +1,8 @@
+#lang racket
+
+(provide square-root)
+
+(define (square-root radicand)
+    (for/first ([num (in-naturals)]
+                #:when (equal? (expt num 2) radicand))
```

Comment:
I think there should be a `#:break` here for when `num` gets too large.
I know the test suite only covers radicands that are perfect squares, but it bothers me that it'd be easy to accidentally trigger an infinite loop by passing in any other argument.

Reply:
I agree, but the example file isn’t public-facing for a practice exercise.

Reply:
Pasterack is down, but a simple solution would be (in-range radicand) instead of (in-naturals). That forces radicand to be a real number. If it’s a float, a #f would be returned at the end of our finite iterations. Optionally, a contract could force radicand to be an integer so #f is then only returned for natural radicands that don’t have natural square roots. That’s fairly Racket-y.

Reply:
I tend to check example solutions on github after I complete an exercise on the website. I understand that in general they are treated as internal proofs that there exists a solution so there's no requirement other than that the tests pass, but I usually find them to be good sources of idiomatic code. That's why I decided to comment above.

Reply:
Tests pass locally on 8.10 CS so I'll troubleshoot the CI later this week.